feat: added CatalogMainSection

AlinaPalyvoda opened the pull request:
- JSX,
- styles

AlinaPalyvoda committed:
feat: added CatalogMainSection

Ahmed-Fahmy-Dev commented on client/src/pages/CataloguePage/CataloguePage.js:
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
+import CatalogMainSection from '../../components/CatalogMainSection/CatalogMainSection';
```
we created an index.js file in the components folder where we export from it all our components. please adjust accordingly.

AlinaPalyvoda replied:
done

Ahmed-Fahmy-Dev commented on client/src/components/CatalogMainSection/CatalogMainSection.js:
```diff
@@ -0,0 +1,33 @@
+      <Container>
+        <Typography
+          variant="h1"
+          color="#FFFFFF"
```
please use our theme for the color value since we have it in our default theme and it is accessible within mui components.

AlinaPalyvoda replied:
done

Ahmed-Fahmy-Dev commented on client/src/components/CatalogMainSection/CatalogMainSection.module.scss:
```diff
@@ -0,0 +1,21 @@
+$mobile: 428 px;
+$tablet: 768px;
+$laptop: 1024px;
+$desktop: 1360px;
+
+$primaryMainColor: #0499dd;
```
you dont need those. they are already available in our mui theme created and accessible through the sx prop

AlinaPalyvoda replied:
done

Ahmed-Fahmy-Dev commented on client/src/components/CatalogMainSection/CatalogMainSection.js:
```diff
@@ -0,0 +1,33 @@
+
+function CatalogMainSection() {
+  return (
+    <Stack className={styles.mainSection} justifyContent="center">
```
you dont need the stack component here as you dont have a stack of elements. you can substitute it with the Box element which by default also translates into a div HTML tag.

AlinaPalyvoda replied:
I used Stack for text positioning convenience. But your comment makes sense. Done

Ahmed-Fahmy-Dev commented on client/src/components/CatalogMainSection/CatalogMainSection.js:
```diff
@@ -0,0 +1,33 @@
+            },
+          }}
+        >
+          OUR TOURS
```
please user text-transform: uppercase; instead of writing uppercase words in JSX

AlinaPalyvoda replied:
Done

Ahmed-Fahmy-Dev reviewed (commented):
- please move the jpg file to this path : public/assets/images/catalogueMainSection

AltysS commented on client/src/components/CatalogMainSection/CatalogMainSection.module.scss:
```diff
@@ -0,0 +1,21 @@
+$primaryMainColor: #0499dd;
+
+.mainSection {
+  background: center url('./catalogMainSectionBG.jpg') no-repeat;
```
I guess it`s better to move image to the following path : client/public/assets/images/ Your folder here / Your Image here.

@Ahmed-Fahmy-Dev , please advise if I`m right here with the file structure.

Ahmed-Fahmy-Dev replied:
absolutely write.   @AlinaPalyvoda  please act accordingly.

AlinaPalyvoda replied:
Done

AlinaPalyvoda committed:
Merge branch 'develop' into feature/catalog-page-main-section

AlinaPalyvoda committed:
feat: catalog page main section:

- component added to index.js
- styling with the styled() utility
- scss removed
- change image path

AlinaPalyvoda committed:
Merge branch 'develop' into feature/catalog-page-main-section

Ahmed-Fahmy-Dev commented on client/src/components/CatalogMainSection/CatalogMainSection.js:
```diff
@@ -0,0 +1,48 @@
+  fontWeight: 700,
+  fontsize: 36,
+  marginBottom: 0,
+  color: 'white',
```
```suggestion
  color: 'theme.palette.primary.contrastText',
```

AlinaPalyvoda replied:
Thanks! Done

Ahmed-Fahmy-Dev reviewed (commented):
great job !!!!!!!!!

AlinaPalyvoda committed:
fix: applied text color from theme

AltysS reviewed (approved):
Well done
